src/app/base/components/Header/Header.tsx

```diff
@@ -155,7 +165,20 @@ const generateItems = (
     }),
     isSelected: isSelected(path, link),
     key: link.url,
-    label: link.label,
+    label:
+      link.label === "Controllers" && vaultIncomplete ? ( // check if vault is set up on all controllers
+        <>
+          <Icon
+            className="p-navigation--item-icon"
+            data-testid="warning-icon"
+            name="security-warning-grey"
+          />
```

Comment:
This isn't purely decorative and conveys important information, for that reason it shouldn't be ignored by assistive technology. You can fix this by adding `aria-label="warning"`.

Reply:
Can do yeah. Only reason I didn't was because of the test mentioned below - means I'm looking for a link with name "warning Controllers", but I'll implement it and update the test to account.

Reply:
There's another technique which you could try and might be more appropriate here - keeping the icon hidden and use `aria-description`/`aria-describedby` on the link instead. Then you couldn test for the description on the link element using `expect().toHaveAccessibleDescription()`. That way the link itself would remain as `Controllers`.